socs/agents/hwp_gripper/agent.py

```diff
@@ -198,6 +239,14 @@ def inp(self, session, params=None):
         **Task** - Queries whether the actuators are in a known position. This
         tells you whether the windows software has detected that the actuator
         has been homed.
+
+        Notes:
+            The most recent data collected is stored in session data in the
+            structure:
+
+                >>> response.session['response']
+                {'result': True,
+                 'log': []}
```

Comment:
Does any logging happen for the INP command? Or does this always come back empty?

Reply:
It always comes back empty